docs/examples/website-change.md

```diff
@@ -0,0 +1,176 @@
+2. Get the saved headers variables from the previous execution of this workflow.
+3. Compare the current and previous headers to determine if the target website has changed.
+4. Save current headers to workflow variables.
+
+Below we'll explain each step in more detail. For this example we've chosen `https://docs.direktiv.io` as the target website.
+
+## Fetch Website Headers
+Here we are calling a HTTP `POST` request on the target website using our `vorteil/request:v5` direktiv app as a function. Since we only care about the `Last-Modified` and `Etag` headers we then extract those values and store them in the `lastModified` and `etag` properties using the transform field in the `fetch-site-headers` state. Now that we have fetched the current header values required, the state will transition to the `get-old-headers` state.
+
+```yaml
+id: check-website-change
+description: "A simple workflow that fetches current headers from a website and compares them to the previously stored headers to determine if it has changed."
+functions:
+  - id: get
+    image: vorteil/request:v5
+states:
+  - id: fetch-site-headers
+    type: action
+    transform: "{lastModified: .return.headers.[\"Last-Modified\"][0], etag: .return.headers.[\"Etag\"][0]}"
+    transition: get-old-headers
+    action:
+      function: get
+      input: |
+        {
+          "method": "HEAD",
+          "url": "https://docs.direktiv.io",
+        }
+```
+
+## Get Old Headers
+Direktiv has variables scoped to namespaces, workflows and instance. In this state (`get-old-headers`) we'll get the variables `lastModified` and `etag` in the workflow scope. These variables are set to whatever the header values were last time when this workflow was executed. If this is the first time this workflow is executed these value will be `null`. This is fine, but it will cause the results to be `siteChanged: true` because the current headers can never equal `null`.
+
+```yaml
+  - id: get-old-headers
+    type: getter
+    transition: check-site
+    variables:
+      - key: lastModified
+        scope: workflow
+      - key: etag
+        scope: workflow
+```
+
+## Compare Values
+Now that we both the current and previous header values we can make a comparison and check whether the website has changed using the switch state `check-site`. 
+
+The switch state below has three possible conditions. The first condition is used for validation and will `transition` to the error state `unsupported-site` if neither `etag` or `lastModified` was fetched from the current headers. The last two are to check if either of the `etag` or `lastModified` values have changed between the previous and current headers. If either one of these headers has changed it means that the website has changed and the property `siteChanged` is set to true. If none of these conditions are satisfied,  the `siteChanged` property is set to false because we can assume that no errors/changes have occurred.
```

Comment:
> If either one of these headers has changed ...

I *think* this should be "If either one of these headers **have** changed", but I'm not 100% sure of that. You might be right.